static/app/views/settings/projectPerformance/projectPerformance.tsx

```diff
@@ -180,6 +204,28 @@ class ProjectPerformance extends AsyncView<Props, State> {
             )}
           </Access>
         </Form>
+        <Feature features={['organizations:performance-issues']}>
+          <Form
+            saveOnBlur
+            allowUndo
+            initialData={{
+              performanceIssueCreationRate:
+                this.state.project.performanceIssueCreationRate,
+            }}
+            apiMethod="PUT"
+            apiEndpoint={projectEndpoint}
+          >
```

Comment:
if I'm reading this correctly, on `save` a `PUT` request will be fired off to update the settings?

Reply:
Yep, it will do a partial update.

Reply:
👍